datacats/environment.py

```diff
@@ -607,11 +608,12 @@ def create_admin_set_password(self, password):
                 'sysadmin': True},
                 out)
         self.run_command(
-            command=['/bin/bash', '-c', '/usr/lib/ckan/bin/ckanapi '
-                     'action user_create -i -c /project/development.ini '
-                     '< /input/admin.json'],
+            command=['/scripts/update_add_admin.sh'],
             db_links=True,
-            ro={self.sitedir + '/run/admin.json': '/input/admin.json'},
+            ro={
+                UPDATE_ADD_ADMIN: '/scripts/update_add_admin.sh',
+                self.sitedir + '/run/admin.json': '/input/admin.json'
+               },
```

Comment:
this can use self.user_run_script now which makes sure to run as a non-root user.